<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>534</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="eventsLayout"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    padding: 4px 8px; /* Ample space */
    border: none;
    border-radius: 4px;
    font-size: 14px;
    min-width: 80px; /* Ensures text fits */
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton:pressed { background-color: #3d8b40; }</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
